rancher2/resource_rancher2_user.go

```diff
@@ -191,6 +191,6 @@ func userStateRefreshFunc(client *managementClient.Client, userID string) resour
 			}
 			return nil, "", err
 		}
-		return obj, obj.State, nil
+		return obj, "active", nil
```

Comment:
seems to me that it's not related to rke1 removal.
maybe i am wrong here but in case it's not related to rke1 removal it would be good to add reason for this in the PR description.

Reply:
Oops...lemme remove that, that's for https://github.com/rancher/terraform-provider-rancher2/issues/1688

Reply:
Ahh...of course, I can't remove that because the user dropped `.State` too.

Reply:
cool seems fine.